socorro/cron/jobs/bugzilla.py

```diff
@@ -36,6 +36,35 @@
 )
 
 
+def find_signatures(content):
+    """Return a list of signatures found inside a string.
+
+    Signatures are found between `[@` and `]`. There can be any number of them
+    in the content.
+
+    Example:
+    >>> find_signatures("some [@ signature] [@ another] and [@ even::this[] ]")
+    set(['signature', 'another', 'even::this[]'])
+    """
+    if not content:
+        return set()
+
+    signatures = set()
+    parts = content.split('[@')
+    # The first item of this list is always not interesting, as it cannot
+    # contain any signatures. We thus skip it.
```

Comment:
Ahh... I see. If `[@` is at the beginning of the string, then the first item is an empty string.